zlib/PSPBUILD

```diff
@@ -9,35 +9,27 @@ license=('ZLIB')
 depends=()
 makedepends=()
 optdepends=()
-source=(
-    "https://sourceforge.net/projects/libpng/files/${pkgname}/${pkgver}/${pkgname}-${pkgver}.tar.gz"
-    "${pkgname}-${pkgver}-PSP.patch"
-)
-sha256sums=(
-    '36658cb768a54c1d4dec43c3116c27ed893e88b02ecfcb44f2166f9c0b7f2a0d'
-    'SKIP'
-)
-
-prepare() {
-    cd "$pkgname-$pkgver"
-    patch -p1 < ../${pkgname}-${pkgver}-PSP.patch
-}
+source=("git+https://github.com/madler/zlib.git#commit=cacf7f1d4e3d44d871b605da3b647f07d718623f")
```

Comment:
Please use the upstream tar.gz if possible

Reply:
done